naps/soc/platform/zynq/memory_accessor_devmem.py

```diff
@@ -33,4 +33,4 @@ def write(self, offset, to_write):
         return self.mem.write(struct.pack('I', to_write))
 
 
-MemoryAccessor = DevMemAccessor
```

Comment:
The `DevMemAccessor` constructor already rounds to `mmap.PAGESIZE` so it's unnecessary to do it again here.

But the idea of textually replacing source code is a little gross to me, maybe this line should be ditched entirely.